launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -235,6 +235,13 @@
     </include>
   </group>
 
+  <group if="$(eval &quot;'$(var objects_validation_method)'=='occupancy_grid'&quot;)">
+    <include file="$(find-pkg-share detected_object_validation)/launch/occupancy_grid_based_validator.launch.xml" if="$(var use_validator)">
```

Comment:
[Q] If this line is added, no validator is launched unless "objects_validation_method == occupancy_grid", so detection pipeline stops here.
Why do you want to add this?